help/destinations/catalog/analytics/gainsight-px.md

```diff
@@ -0,0 +1,107 @@
+* **[!UICONTROL Username]**: The email used to login to the [Gainsight PX UI](https://app.aptrinsic.com)
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![destination details screenshot](../../assets/catalog/analytics/gainsight_px/destination_details.png)
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>* To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map attributes and identities {#map}
+
+The only relevant mapping is between the Adobe identity and the Gainsight PX identity.  Choose the appropriate identity source field on the Adobe side and map it to the IDENTIFY_ID (the only choice) field on the target side.  It is important that the source field have the exact same value that is passed during the Gainsight PX identify step.
+![Example showing the identity mapping](../../assets/catalog/analytics/gainsight_px/mapping_identities.png)
+
+## Exported data / Validate data export {#exported-data}
+
+Segmentation data is streamed from the Adobe environment to Gainsight PX.  
+Segment metadata will be visible in the Segments screen within the PX application:![Segments screen](../../assets/catalog/analytics/gainsight_px/segment_metadata.png)  
+Segment membership information will be visible on the Segments tab of the Audience Explorer screen within PX: ![Audience Explorer](../../assets/catalog/analytics/gainsight_px/PX_Segments.png)
```

Comment:
Segment membership information will be visible on the Segments tab of the Audience Explorer screen in the [!DNL Gainsight PX] UI.
![Audience Explorer]